tests/test_projects/contract_id_type/Forc.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "contract_id_type"
+
+[dependencies]
+core = { git = "http://github.com/FuelLabs/sway-lib-core" }
```

Comment:
Does this actually use `core`? If not, just remove.

Reply:
done